ui/src/hooks/useTerminalConnection.ts

```diff
@@ -393,7 +393,7 @@ export const useTerminalConnection = (lunaId: Ref<string>, origin: Ref<string>)
 
       sendEventToLuna('KEYBOARDEVENT', '');
 
-      socket.send(formatMessage(terminalId.value, FormatterMessageType.TERMINAL_DATA, processedData));
+      socket.send(formatMessage(terminalId.value, FORMATTER_MESSAGE_TYPE.TERMINAL_DATA, processedData));
     });
   };
 
```

Comment:
The code is not providing any information that can be analyzed or used to make recommendations for improvement. The content is simply repeating sections of TypeScript code without additional context or analysis of its structure, purpose, and function. To provide feedback on specific aspects of this code, please include more detailed comments discussing the functionality and intended use of each part.